Assets/Sanji.cs

```diff
@@ -13,6 +14,8 @@ public class Sanji : Character {
 
 		player.knockbackVector = new Vector2 (direction > 0 ? -1 : 1, 1);
 		player.wasThrown (direction);
+
+		hitter.GetComponent<Health> ().Damage (kickDmg);
```

Comment:
Ah good point. I was starting to think of PlayerMovement more as just "Player". It does only have movement though, so we should keep it that way (it also has animation and player control, but that also happens to be closely related to PlayerMovement, so that is fine being all in there).